ndcube/tests/helpers.py

```diff
@@ -95,9 +96,11 @@ def assert_metas_equal(test_input, expected_output):
             assert test_input[key] == expected_output[key]
 
 
-def assert_cubes_equal(test_input, expected_cube):
+def assert_cubes_equal(test_input, expected_cube, check_data=False):
```

Comment:
Why make checking data an option?  Why not always do it?

Reply:
Depending on the size of the data, this could slow down the unit test, especially if do not care if the data changed (or not).

Reply:
Let's make the default of `check_data=True` then.